test_scripts/Defects/8_1/1883_1_PerformInteraction_REJECTED_for_being_deleted_InteractionChoceSet.lua

```diff
@@ -0,0 +1,108 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1883
+---------------------------------------------------------------------------------------------------
+-- Description: Check SDL rejects PerfromInteraction request if it's being sent during deleting of InteractionChoiceSet
+---------------------------------------------------------------------------------------------------
+-- Preconditions:
+-- 1. SDL and HMI are started
+-- 2. Mobile app is registered and activated
+-- In case:
+-- 1. App is created InteractionChoiseSet(choiceSetID_1)
+-- 2. App sends DeleteInteractionChoiceSet(choiceSetID_1) request
+-- SDL does:
+-- - transfer request to HMI
+-- 3. App sends PerformInteraction(choiceSetID_1) request
+-- SDL does:
+-- - not transfer request to HMI
+-- - responds with REJECTED to the App
+-- 4. HMI responds to DeleteInteractionChoiceSet
+-- SDL does:
+-- - responds DeleteInteractionChoiceSet(SUCCESS) to App
```

Comment:
```suggestion
-- - respond DeleteInteractionChoiceSet(SUCCESS) to App
```
Check for all scripts

Reply:
Done